scripts/us_census/acs5yr/subject_tables/common/generateColMap.py

```diff
@@ -0,0 +1,94 @@
+"""Module to generate a column map to their corresponding statistical variable defintions"""
+
+import os
+import json
+import pandas as pd
+
+#TODO:Remove this import and make it from ~/data/utils/ directory when it is available
+os.system('curl https://raw.githubusercontent.com/Abilityguy/data/dcid_gen/util/statvar_dcid_generator.py -LO statvar_dcid_generator.py')
+import statvar_dcid_generator
+
+dcid_gen = statvar_dcid_generator.StatVarDcidGenerator()
+
+class generateColMapBase:
+  """module to generate a column map given a list of columns of the dataset and a JSON Spec"""
+  def __init__(self, specDict=None, columnList=None):
+    """module init """
+    self.features = specDict
+    self.columnList = columnList
+    self.columnMap = {}
+    self.makeColMap()
+
+  def makeColMap(self):
```

Comment:
nit: this module can perhaps export a single function with arguments, and this class can be local to this module.  So the call-site can be:

`thing_generator.generate(...)` instead of `thing_generator.generateClass(...).generate()`

Reply:
Thank you for the suggestion, I have implemented along similar lines, please let me know if it is close to how we want this?